### PR TITLE
Feature 82

### PR DESCRIPTION
**Pull request by ryleyw**
Needs review for the implementation of react navbar and resource page

### CODE REVIEW

File: react-frontend/src/components/Nav/footer.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import '../../css/footer.css';
+import { Link } from 'react-router-dom';
 function Footer() {
   return (
-    <footer class="footer">
-      <div class="container">
+    <footer className="footer">
+      <div className="container">
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Nav/footer.js:7]: is this container class used for anything?  wondering if we need it at all or if the footer does the job?

File: react-frontend/src/components/Nav/navbar.js [hunk 2]
```diff
@@ -1,11 +1,50 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import '../../css/nav.css';
+import { Link, useLocation, useHistory } from 'react-router-dom';
+import '../../font-awesome/css/font-awesome.min.css';
 
 function NavBar() {
+  const history = useHistory();
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Nav/navbar.js:7]: I'm kind of curious what this does/is used for

File: react-frontend/src/components/Resources/digitalcards.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  'Read the principals',
+  'Read the policy',
+];
+
+function DigitalCards() {
+  return (
+    <div className="digitalCards">
+      <Grid className="cardGrid">
+        <Row>
+          <Col xs={12} sm={12} md={12} lg={4}>
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Resources/digitalcards.js:28]: don't need all these size conditions

File: react-frontend/src/components/Resources/fordesigners.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import { Col, Row, Grid } from 'react-flexbox-grid';
+import '../../css/resources.css';
+
+function ForDesigners() {
+  return (
+    <div className="forDesigners">
+      <div className="resourceInfoTitle">For Designers</div>
+      <Grid>
+        <Row>
+          <Col xs={12} sm={12} md={12} lg={4}>
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Resources/fordesigners.js:11]: don't need all these sizes

File: react-frontend/src/components/Resources/fordevelopers.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import InfoCard from '../Home/infocard';
+
+function ForDevelopers() {
+  return (
+    <div className="forDevelopers">
+      <div className="resourceInfoTitle">For Developers</div>
+      <div className="resourceInfo">
+        <Grid>
+          <Row>
+            <Col xs={12} sm={12} md={6} lg={6}>
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Resources/fordevelopers.js:12]: same as above

File: react-frontend/src/components/Resources/standards.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+import '../../css/resources.css';
+
+function Standards() {
+  return (
+    <div className="standards">
+      <div className="resourceInfoTitle">Standards & Policy</div>
+      <div className="resourceInfo">
+        <Grid>
+          <Row>
+            <Col xs={12} sm={12} md={12} lg={4}>
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Resources/standards.js:12]: same as above

File: react-frontend/src/containers/App/index.js [hunk 7]
```diff
@@ -2,16 +2,26 @@ import React from 'react';
 //import Nav from '../../components/Nav';
 import NavBar from '../../components/Nav/navbar.js';
 import Home from '../../components/Home/home.js';
+import Resources from '../../components/Resources/resources.js';
```
Comment (thegentlemanphysicist) [inline react-frontend/src/containers/App/index.js:5]: imports don't need the .js  in react

File: react-frontend/src/font-awesome/HELP-US-OUT.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
```
Comment (thegentlemanphysicist) [inline react-frontend/src/font-awesome/HELP-US-OUT.txt:1]: Is there a reason this file is included?  If not, I would suggest deleting.

Comment (thegentlemanphysicist) [inline react-frontend/src/font-awesome/HELP-US-OUT.txt:1]: I don't think we want 'font-awesome' installed in the src of the project. Though that's more of an intuition than anything concrete.  Is it possible to import it as a package?

File: react-frontend/package.json [hunk 9]
```diff
@@ -14,6 +14,7 @@
     "react-apollo": "^3.1.5",
     "react-dom": "^16.13.1",
     "react-flexbox-grid": "^2.1.2",
+    "react-head": "^3.3.0",
```
Comment (thegentlemanphysicist) [inline react-frontend/package.json:17]: Is this used anywhere?  I could not find it declared anywhere.

File: react-frontend/src/components/Nav/navbar.js (hunk 2 above)
Comment (ryleyw) [reply react-frontend/src/components/Nav/navbar.js:7]: When route changes, usehistory is used to fetch the current route name and change the current page. Highlighting in navbar can then watch this state and update itself accordingly

File: react-frontend/package.json (hunk 9 above)
Comment (ryleyw) [reply react-frontend/package.json:17]: Might need for the icon import, will update this when I get it figured out

Comment (ryleyw) [reply react-frontend/package.json:17]: Don't need it, removing

File: react-frontend/src/font-awesome/HELP-US-OUT.txt (hunk 8 above)
Comment (ryleyw) [reply react-frontend/src/font-awesome/HELP-US-OUT.txt:1]: removed font-awesome css/fonts

**Pull request merged**